src/classes/contexts/CommandContext.ts

```diff
@@ -61,23 +61,47 @@ export class CommandContext implements CommandContextBuilder {
 	public async react(emoji: EmojiIdentifierResolvable) {
 		return await this.message.react(emoji);
 	}
+	
+	public async reacts(emojis: EmojiIdentifierResolvable[]) {
+		for (const emoji of emojis) {
+			await this.message.react(emoji);
+		}
+	}
 
-	public async removeReactions() {
+	public async removeReactionsAll() {
 		return this.message.reactions.removeAll();
 	}
 
 	public async removeReaction(emoji: EmojiIdentifierResolvable) {
 		return this.message.reactions.resolve(typeof emoji === 'object' ? emoji.id! : emoji)!.remove();
 	}
+	
+	public async removeReactions(emojis: EmojiIdentifierResolvable[]) {
+		for (const emoji of emojis) {
+			this.message.reactions.resolve(typeof emoji === 'object' ? emoji.id! : emoji)!.remove();
+		}
+	}
 
 	public async removeSelfReaction(emoji: EmojiIdentifierResolvable) {
 		return this.message.reactions.resolve(typeof emoji === 'object' ? emoji.id! : emoji)!.users.remove(this.client.user!.id);
 	}
+	
+	public async removeSelfReactions(emojis: EmojiIdentifierResolvable[]) {
+		for (const emoji of emojis) {
+			this.message.reactions.resolve(typeof emoji === 'object' ? emoji.id! : emoji)!.users.remove(this.client.user!.id);
+		}
+	}
 
 	public deleteMessage(timeout: number = 0) {
 		return this.message.delete({timeout});
 	}
 
+	public bulkDeleteChannel(number: number | Collection<string, Message> | readonly MessageResolvable[], filterOld?: boolean | undefined) {
```

Comment:
I think that it should be renamed `bulkDeleteMessages`, otherwise it could make it seem like it is for deleting channels 👀